lib/utils/resolveSilent.mjs

```diff
@@ -0,0 +1,46 @@
+import fs from 'node:fs';
+import path from 'node:path';
+
+import { fileURLToPath, pathToFileURL } from 'node:url';
+
+import { moduleResolve } from '@dual-bundle/import-meta-resolve';
+import resolveFrom from 'resolve-from';
+
+const pathSuffixes = ['', '.js', '.json', `${path.sep}index.js`, `${path.sep}index.json`];
```

Comment:
minor
question

We don't need to support `.cjs` nor `.mjs` I suppose?

Reply:
Those two suffixes must be used specifically, and these suffixes are only here for backward compatibility.